.circleci/config.yml

```diff
@@ -508,6 +516,19 @@ jobs:
             - check-public-documentation
       - run: exit 0
 
+  publish-documentation:
+    executor: ndk-r21e-latest-executor
+    steps:
+      - checkout
+      - prepare-mbx-ci
+      - run:
+          name: Publish API docs Android
+          command: |
+            GITHUB_WRITER_TOKEN=$(./mbx-ci github writer token)
```

Comment:
We have to use `mbx-ci github writer public token` now. Details here: #5233.

Reply:
nice one! thanks

Reply:
fixed